awkward-cpp/pyproject.toml

```diff
@@ -0,0 +1,58 @@
+[build-system]
+requires = [
+	"scikit-build-core @ git+https://github.com/scikit-build/scikit-build-core@887757e85d728bad97fb883c0cf7c132298e32f6",
+    "pybind11",
+    "ninja==1.10.2.4; platform_system!='Windows'",
+    "cmake"
```

Comment:
```suggestion
```

You should never specify cmake and ninja when using scikit-build-core, it will automatically add those only if the system versions are inadequate. This way you can build on systems that do not provide pre-built wheel!